Vuzix/app/src/androidTest/java/com/facilitation/view/ExampleInstrumentedTest.kt

```diff
@@ -1,24 +0,0 @@
-package com.facilitation.view
```

Comment:
Do you think there are some tests that could be written for this?

Reply:
I have one but might need to rearrange some formatting for the next task so I will finish the tests with that task